### ISSUE TITLE
Handle the "unsure" annotations

### ISSUE THREAD

**Issue by thomashopkins32** [thomashopkins32/HuBMAP#5]
We also receive annotations that the experts who annotated the data are unsure about.

We could try the following:
- Treat them as full labels
- Treat them as full labels but with less of a penalty in the loss for getting them wrong (tunable)
- Smooth the labels to be `0.5` for these masks

Each of these options should be tested and examined in isolation on various sizes of the training set (probably need cross validation).

**Comment by thomashopkins32** [comment]
It looks like some annotations only have "unsure" as the label. This means that we should definitely be including it in our training somehow.
